modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1105,5 +1111,23 @@ class Edit_User extends \NDB_Form
         }
         return $permission;
     }
+
+    /**
+     * Gathers JS dependencies and merges them with the parent
+     *
+     * @return array of javascript to be inserted
+     */
+    function getJSDependencies()
+    {
+        $factory = \NDB_Factory::singleton();
+        $baseURL = $factory->settings()->getBaseURL();
+        $deps    = parent::getJSDependencies();
+        return array_merge(
+            $deps,
+            array(
+             $baseURL . "/user_accounts/js/rejectUser.js"
+            )
+        );
+    }
 }
-?>
\ No newline at end of file
+?>
```

Comment:
can someone tell me what this change means/does?

Reply:
@nicolasbrossard can as I asked him this same question a few days ago!

Reply:
https://stackoverflow.com/questions/19953483/php-end-tag

Reply:
It ends the PHP (started with `<?php`) so that anything outside of it is treated as a literal and just sent to the browser as is.

Modern PHP standards generally say to not include the end tag, to prevent problems with accidentally sending whitespace to the browser..